dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/validation/hooks/EventDataValuesValidationHook.java

```diff
@@ -111,20 +105,21 @@ private void validateMandatoryDataValues( Event event, TrackerImportValidationCo
         }
     }
 
-    private void validateDataElement( ValidationErrorReporter reporter, DataElement dataElement,
+    private void validateDataValue( ValidationErrorReporter reporter, DataElement dataElement,
         DataValue dataValue, ProgramStage programStage, Event event )
     {
-        final String status = ValidationUtils.dataValueIsValid( dataValue.getValue(), dataElement );
+        final ErrorMessage errorMessage = valueTypeValidationUtils.dataValueIsValid( dataElement,
+            dataValue.getValue() );
 
-        if ( status != null )
+        if ( errorMessage != null )
         {
-            reporter.addError( event, TrackerErrorCode.E1302, dataElement.getUid(),
-                status );
+            reporter.addError( event, getTrackerErrorCode( errorMessage ),
+                dataElement != null ? dataElement.getUid() : null,
+                errorMessage.getMessage() );
```

Comment:
It looks like you are passing the error message as an argument to the report (not only here). This would mean the string value of the error message gets interpolated as an arg in the tracker error message. Can you please check if the error message is rendered as you would expect it.